package.json

```diff
@@ -74,7 +74,6 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "randomcolor": "^0.5.3",
-    "rusha": "^0.8.6",
```

Comment:
Move rusha to peerDependencies

Reply:
Is this comment still relevant if I'm using the rusha file as worker?